controllers/reconcilers/logging_installation/logging_installation_reconciler.go

```diff
@@ -0,0 +1,283 @@
+package logging_installation
+
+import (
+	"context"
+	"strings"
+
+	"github.com/go-logr/logr"
+	v14 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	v1 "github.com/redhat-developer/observability-operator/v3/api/v1"
+	"github.com/redhat-developer/observability-operator/v3/controllers/model"
+	"github.com/redhat-developer/observability-operator/v3/controllers/reconcilers"
+	v12 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type Reconciler struct {
+	client client.Client
+	logger logr.Logger
+}
+
+func NewReconciler(client client.Client, logger logr.Logger) reconcilers.ObservabilityReconciler {
+	return &Reconciler{
+		client: client,
+		logger: logger,
+	}
+}
+func (r *Reconciler) Cleanup(ctx context.Context, cr *v1.Observability) (v1.ObservabilityStageStatus, error) {
+	if cr.DescopedModeEnabled() {
+		return v1.ResultSuccess, nil
+	}
+
```

Comment:
Should the ClusterLoggings and ClusterLogForwarders CRs created by OO be removed during Cleanup?

Reply:
Good point, I've added those resources to the cleanup